recipes/minimp3/all/test_package/test_package.c

```diff
@@ -0,0 +1,9 @@
+#define MINIMP3_IMPLEMENTATION
```

Comment:
is this macro required to use the library? if so, it should go into `package_info` (**self.cpp_info.defines**)

Reply:
No, it must defined in only one compilation unit.